Makefile

```diff
@@ -18,7 +18,7 @@ GRAVITY_PKG_PATH ?= github.com/gravitational/gravity
 ASSETSDIR=$(TOP)/assets
 BINDIR ?= /usr/bin
 
-# Current Kubernetes version: 1.13.0
+# Current Kubernetes version: 1.13.3
 K8S_VER := 1.13.0
```

Comment:
Will update this when https://github.com/gravitational/planet/pull/389 is merged and planet is tagged appropriately.